src/test/java/seedu/address/logic/commands/EditModuleClassCommandTest.java

```diff
@@ -159,6 +159,9 @@ public void equals() {
         // null -> return false
         assertFalse(standardCommand.equals(null));
 
+        // different types -> return false
+        assertFalse(standardCommand.equals(new ClearModuleClassCommand()));
+
```

Comment:
Is there any particular reason why we choose `ClearModuleClassCommand` as the different type as opposed to some other primitive type like an `integer`?

Reply:
That test was missing so i just added it and made it consistent with what was in EditStudentClassCommandTest